File: SkyEditor.Core.Windows/Extensions/Online/RootCollectionResponse.vb
Namespace Extensions.Online
    Public Class RootCollectionResponse
        Public Property Name As String
        Public Property ChildCollections As List(Of ExtensionCollectionModel)
        Public Property ExtensionCount As Integer
        Public Property GetExtensionListEndpoint As String
        Public Property DownloadExtensionEndpoint As String
    End Class

End Namespace

File: SkyEditor.Core/Extensions/Online/RootCollectionResponse.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SkyEditor.Core.Extensions.Online
{
    /// <summary>
    /// The response of an extension server
    /// </summary>
    public class RootCollectionResponse
    {
        public string Name { get; set; }
        public List<ExtensionCollectionModel> ChildCollections { get; set; }
        public int ExtensionCount { get; set; }
        public string GetExtensionListEndpoint { get; set; }
        public string DownloadExtensionEndpoint { get; set; }
    }
}
